database/migrations/2020_01_12_000000_create_shoppingcart_table.php

```diff
@@ -11,20 +11,21 @@ class CreateShoppingcartTable extends Migration
      */
     public function up()
     {
-        Schema::create('shoppingcart', function (Blueprint $table) {
+        Schema::create('shopping_carts', function (Blueprint $table) {
+            $table->bigIncrements('id');
             $table->string('identifier');
             $table->string('instance');
```

Comment:
This might also need to be indexed ?

Reply:
It does, but the subsequent `$table->unique(['identifier', 'instance']);` covers it.